src/main/kotlin/com/sourcegraph/cody/CodyToolWindowContent.kt

```diff
@@ -54,10 +58,11 @@ class CodyToolWindowContent(private val project: Project) : UpdatableChat {
       JButton("Stop generating", IconUtil.desaturate(AllIcons.Actions.Suspend))
   private val recipesPanel: JBPanelWithEmptyText
   val embeddingStatusView: EmbeddingStatusView
-  override var isChatVisible = false
-  override var id: String? = null
+  var isChatVisible = false
+  var id: String? = null // todo it's de facto historyservice.activeId, so we should remove that
   private var codyOnboardingGuidancePanel: CodyOnboardingGuidancePanel? = null
   private val chatMessageHistory = CodyChatMessageHistory(CHAT_MESSAGE_HISTORY_CAPACITY)
```

Comment:
`chatMessageHistory` does not work when changing b/w chats - idk if its required feature